components/toast/Toast.js

```diff
@@ -54,24 +53,16 @@ class Toast extends PureComponent {
   };
 
   renderCustomLink = () => {
-    const { link, linkLabel, linkTarget } = this.props;
-    return (
-      link && (
-        <TextBody className={theme['label']} color="white" soft>
-          <Link href={link} target={linkTarget} className={theme['action-link']} inherit>
-            {linkLabel}
-          </Link>
-        </TextBody>
-      )
-    );
+    const { link } = this.props;
+    return link && <TextBody color="white">{React.cloneElement(link, { className: theme['toast-link'] })}</TextBody>;
```

Comment:
* I guess you can remove the `color="white"` here, since you change the color with css as well.
* Not 100% sure, but I think the original `className` will be overwritten from the link element (if it has one). You can probably change it to `cx(link.props.className, theme['toast-link'])`

Reply:
I tested the second thing and that was correct. Updated the code in the latest commit.

 :ok: by @Kemosabert on 28/06/2018 17:12:53